snapcraft/main.py

```diff
@@ -110,6 +113,8 @@ def main():
 
     log.configure(log_level=log_level)
 
+    common.set_enable_parallel_builds(not args['--no-parallel-build'])
+
     try:
         commands.load(cmd).main(argv=args['ARGS'])
```

Comment:
Sure, that's a great point. Let's just keep thinking about how to get rid of all the globals.